src/ChirpWeb/Pages/Public.cshtml

```diff
@@ -30,4 +34,15 @@
     {
         <em>There are no cheeps so far.</em>
     }
+    <button class="button" onclick="navigateToPage(@previous)" style="color: ECCFC3;" id="previousButton"
```

Comment:
Can this be moved to Layout class? Maybe then we can use it for both public and private